dev/com.ibm.ws.webcontainer.cors/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -15,19 +15,20 @@
 #NLS_MESSAGEFORMAT_NONE
 
 cors=Cross-Origin Resource Sharing
-cors.desc=Refers to a cross-domain security procedure for JavaScript-based clients.
+cors.desc=Configuration for the Cross-Origin Resource Sharing (CORS).
 
 domain=Domain name
 domain.desc=The domain name represents the URL being setup with these CORS settings.
 
 allowedOrigins=Allowed origins
-allowedOrigins.desc=A comma-separated list of origins allowed to access the configured domain.
+allowedOrigins.desc=A comma-separated list of origins allowed to access the configured domain. 
+allowedOrigins.desc=A comma-separated list of origins allowed to access the configured domain. If this is set to "*", it indicates the resource is publicly accessible, with no access control checks. Also, the value of "null" indicates, the domain must not be accessible.  
```

Comment:
allowedOrigins.desc=A comma-separated list of origins **that are** allowed to access the configured domain. 

allowedOrigins.desc=A comma-separated list of origins **that are** allowed to access the configured domain. If this is set to "*", it indicates the resource is publicly accessible, with no access control checks. Also, the value of "null" **indicates** the domain must not be accessible.

Reply:
Just in case it's hard to find, you don't need a , after "null".